ietf/utils/draft.py

```diff
@@ -820,8 +825,9 @@ def dotexp(s):
                     first, last = author.rsplit(".", 1)
                     first += "."
                 else:
-                    author = "[A-Z].+ " + author
-                    first, last = author.rsplit(" ", 1)
+                    # handle single-word names
```

Comment:
Was the `[A-Z].+` doing anything functional before?

Reply:
I don't think so. If we end up in that part of the conditional, `author` does not contain a space, so it is a single word. So we'd never match a first name.

Reply:
It looks like a bug to me. At the very least, the old code was a strange construct and I suspect it is an edit-fail from long ago.

Reply:
Thanks, that's what it looked like but the code is a bit tricky (to say the least).